test/e2e/lb_eip_test.go

```diff
@@ -106,6 +106,10 @@ func TestAWSEIPAllocationsForNLB(t *testing.T) {
 		t.Fatalf("failed to observe expected conditions: %v", err)
 	}
 
+	// Get ELB Hostname via wildcard DNS record for use with verifyExternalIngressController.
```

Comment:
What do you mean by "via wildcard DNS record"?  You're getting the host name from the service's status.  
```suggestion
	// Get ELB host name from the service status for use with verifyExternalIngressController.
```

Reply:
I started with looking up host name via wildcartd DNS Record target, but then came to the conclusion using the service is more direct.

Forgot to update this. Done.